open-media-match/src/OpenMediaMatch/blueprints/hashing.py

```diff
@@ -10,18 +26,65 @@ def hash_media():
     Fetch content and return its hash.
     TODO: implement
     """
+
+    content_type = _parse_request_content_type()
```

Comment:
How about we get the content type from the response headers (below) and simplify the interface?
`download_resp.headers['content-type']`